src/components/ErrorComponent.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+const ErrorComponent = ({ loading, error }) => (
+  <div className="quote">
+    {loading && <span>Loading quote</span>}
+    {error && (
+      <span className="error">
+        Something went wrong and we failed to get the quote
+      </span>
+    )}
+  </div>
+);
```

Comment:
- [ ] Your error state is not being handled properly and as such the error message which you have used here is not being displayed to the user rather the image below is what is being shown when i removed a single letter from the media api. Please kindly handle your errors correctly so it is displayed on the screen for the user :+1: